### DISCUSSION TITLE
chakra-cli tokens error

### DISCUSSION THREAD

**Discussion by zgid123** [chakra-ui/chakra-ui#7541] [Q&A]
Hi,

I am using `chakra-cli tokens` to generate typing for typescript. I am using pnpm monorepo.

When I run it, it will raise error 

```sh
Error: Cannot find module '@swc/helpers/lib/_export_star.js'
```

Then I installed package `@swc/helpers` and run again

```sh
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/_export_star.js' is not defined by "exports"
```

I made it work before by installing `@swc/helpers`, but now it didn't work after I upgraded chakra-ui new version.

**Comment by javadev-jef** [comment]
I'm using npm and got an error similar to @zgid123.

```
Error: Cannot find module 'src/theme/index.ts'
```

**Comment by javadev-jef** [comment]
I found the error in my code. The problem was due to a font import. Example:
```
import '@fontsource/exo-2';
```
